contrib/capnproto/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "capnproto"
+pkgver = "0.10.2"
+pkgrel = 0
+build_style = "gnu_configure"
+hostmakedepends = ["autoconf", "automake", "libtool", "pkgconf"]
```

Comment:
automake pulls autoconf already

although in this case cmake also works so that is probably better

Reply:
switched to cmake